cmd/registry-replacer/main.go

```diff
@@ -0,0 +1,210 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+)
+
+func main() {
+	configDir := flag.String("config-dir", "", "The directory with the ci-operator configs")
+	flag.Parse()
+	if *configDir == "" {
+		logrus.Fatal("--config-dir arg is required")
+	}
+
+	var errs []error
+	errLock := &sync.Mutex{}
+	wg := sync.WaitGroup{}
+	if err := config.OperateOnCIOperatorConfigDir(
```

Comment:
Does this need a concurrency limit _a la_ the other tools with ulimit or whatever

Reply:
I prefer increasing my users ulimit and keeping the tool fast :)